db/seeds.rb

```diff
@@ -36,7 +37,7 @@
 end
 
 # <<---- User seeds ---->>
-user1 = User.create!(email: "a@a.a", password:"123456", username:"user1", location: "Miami")
+user1 = User.create!(email: "a@a.a", password:"123456", username:"user1", location: "Miami", phone_number: "+15142681755")
```

Comment:
do we need to add the new User variable to the sanitized parameters in the Devise / application controller?